app/views/forms/show_live.html.erb

```diff
@@ -5,9 +5,50 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l govuk-!-margin-bottom-2">
-      <span class="govuk-caption-l"><%= @form.name %></span><span class="govuk-visually-hidden"> - </span>
-      <%= @form.live? ? t("forms.form_overview.live_title") : t("forms.form_overview.title") %>
+      <%= @form.name %>
     </h1>
-    <p class="govuk-body">Live form placeholder</p>
+    <dl class="govuk-!-margin-bottom-7">
+      <dt class="govuk-visually-hidden">Status</dt>
+      <dd class="govuk-!-margin-0"><%= render FormStatusTagComponent::View.new(status: @form.status) %></dd>
+    </dl>
+
+    <p class="govuk-body">
+      <%= render PreviewLinkComponent::View.new(@form.pages, link_to_runner(ENV["RUNNER_BASE"], @form.id, @form.form_slug)) %>
+    </p>
+
+    <h2 class="govuk-heading-m"><%= t('show_live_form.form_url') %></h2>
+    <%= render FormUrlComponent::View.new(link_to_runner(ENV["RUNNER_BASE"], @form.id, @form.form_slug, live: true ))%>
+
+    <h2 class="govuk-heading-m"><%= t('show_live_form.form_content') %></h2>
+    <ul class="govuk-list">
+      <li> <p class="govuk-body"><%= govuk_link_to t('show_live_form.questions', question_count: @form.pages.count), form_pages_path(@form.id) %></p> </li>
```

Comment:
Might be useful to pluralise the text?

https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize 

```suggestion
      <li> <p class="govuk-body"><%= govuk_link_to pluralize(@form.pages.count,  t('show_live_form.questions'), form_pages_path(@form.id) %></p> </li>
```